types/types.go

```diff
@@ -304,6 +304,8 @@ type SystemContext struct {
 	RegistriesDirPath string
 	// Path to the system-wide registries configuration file
 	SystemRegistriesConfPath string
+	// If not "", overrides the path to save the credentials file (auth.json) to
+	AuthFilePath string
```

Comment:
`s/save the//; s/ to$//` ?

Reply:
I am not understanding this comment, but I changed the comment for AuthFilePath